modules/Kconfig.cmsis_nn

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020 Silvano Cortesi <cortesis@student.ethz.ch>
+# SPDX-License-Identifier: Apache-2.0
+
+comment "Components"
+
+config CMSIS_NN_ACTIVATION
+	bool "Activation"
+	default y
+	help
+	  This option enables the NN libraries for the activation layers
+	  It can perform activation layers, including ReLU (Rectified
+	  Linear Unit), sigmoid and tanh.
+
+config CMSIS_NN_BASICMATHS
```

Comment:
Why `BASICMATHS` with the `S`? I understand the linguistic preference for an `s` in some contexts, but I was testing this out, trying to put a simple test function together for sanitycheck, and the extra `S` caught me off guard since the CMSIS-NN library uses `basicmath` for the folder, doxygen sections, etc. For example: https://github.com/zephyrproject-rtos/cmsis/pull/6/files#diff-2ce6a9fd5ca36dd23269b984185b588dR40

I think consistency is important here and would suggest `CMSIS_NN_BASICMATH`.

Reply:
This one is a typo, changed that.